getting-started/README.md

```diff
@@ -72,7 +72,7 @@ Compiling a native executable takes a bit longer, as GraalVM performs additional
 steps to remove unnecessary codepaths. Use the  `native` profile to compile a
 native executable:
 
-> ./mvnw package -Dnative
+> ./mvnw package -Dnative -Dquarkus.native.container-build=true
```

Comment:
The flag is only required to build a container-compatible executable.

Reply:
You seem to be wrong here. Omitting the flag requires ```native-image``` being present as executable locally. Comment please in follow-up #442